Tests/PostgresNIOTests/New/PostgresQueryTests.swift

```diff
@@ -79,4 +76,19 @@ final class PostgresQueryTests: XCTestCase {
 
         XCTAssertEqual(query.binds.bytes, expected)
     }
+
+    func testUnsafeSQLInjection() {
```

Comment:
This is just me being silly, but you know, if you're gonna test SQL injection, why not at least have the test do a classical injection? Say, `#"SELECT foo FROM dual LIMIT 0; INSERT INTO users (username, password, isSuperuserOverAllTheInternet) VALUES ("Good ol' Bobby Tables", "easy password", 1); -- PWNED"#` 😇